### PR TITLE
Katerina-Shemet-MiniGame & Homework13

### PR DESCRIPTION
**Pull request by Sedjex**
[map1.txt](https://github.com/BeetrootAcademy-DenysKniaziev/csharp-20221129/files/10350802/map1.txt)   - it is the  map for Pac Man mini game. Put it to bin-Debug-net7.0 folder

Another link: https://drive.google.com/file/d/1AaSrj0sus8nSOunOz02Mq_hJ6r2zXYH8/view?usp=share_link

### CODE REVIEW

File: Katerina Shemet/Lesson13.Homework/Program.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    public IOrderStatus OrderStatus;
+    public Product Product;
+
+    public void Validate()
+    {
+        Console.WriteLine("Validated");
+    }
+}
+
+public interface IOrderStatus
```
Comment (denys-kniaziev) [inline Katerina Shemet/Lesson13.Homework/Program.cs:114]: Dont see any implementation of this interface

**Review by denys-kniaziev — COMMENTED** [review]
Could you please provide map.txt file to test your game?

File: Katerina Shemet/Lesson13.Homework/Program.cs (hunk 1 above)
Comment (Sedjex) [reply Katerina Shemet/Lesson13.Homework/Program.cs:114]: added the implementation of the interface

**Review by denys-kniaziev — APPROVED** [review]
Nice game!

**Pull request merged**